### ISSUE TITLE
Add payment method feature in MYACCOUNT

### ISSUE THREAD

**Issue by sravanyabendi** [Ananya02850/estore#21]
Design the payment method page where in customer can enter debit card, credit card, net banking, etc.,

**Comment by sravanyabendi** [comment]
can you assign me to this issue

**Comment by Ananya02850** [comment]
yes go ahead

**Comment by Ananya02850** [comment]
@sravanyabendi In product details page we need to redirect to corresponding items as indicated. Can you add to redirect when there is a click event on Accessories & kids wardrobe module by adding products on Accessories & Kids wardrobe. Are you ok with it if the issue  be assigned to you?

**Comment by sravanyabendi** [comment]
yeah, but can I make multiple pages for that or should be done in that page itself?

**Comment by Ananya02850** [comment]
@sravanyabendi should be done in that page itself
